dbicdh/_common/upgrade/26-27/005-pick-groups.pl

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2015 SUSE Linux GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#!perl
+
+use strict;
+use warnings;
+
+sub {
+    my $schema = shift;
+
+    my $products = $schema->resultset('Products');
+    while (my $r = $products->next) {
+        my $group = $r->distri;
+        if ($r->distri eq 'sle' && $r->flavor =~ m/^Server/) {
+            $group = 'sles';
+        }
+        if ($r->distri eq 'sle' && $r->flavor =~ m/^Desktop/) {
+            $group = 'sled';
+        }
+        if ($r->version ne '*') {
+            $group .= "-" . $r->version;
+        }
```

Comment:
what I wonder: perhaps we should stop with "openSUSE-13.2" and use space as seperator?